app/viewmodels/tad/posttransition/ConsignmentItemViewModel.scala

```diff
@@ -51,12 +51,16 @@ case class ConsignmentItemViewModel(
 
 object ConsignmentItemViewModel {
 
-  def apply(ie015: CC015CType, houseConsignment: CUSTOM_HouseConsignmentType03, consignmentItem: CUSTOM_ConsignmentItemType03): ConsignmentItemViewModel =
+  def apply(ie015: CC015CType,
+            houseConsignment: CUSTOM_HouseConsignmentType03,
+            hcIndex: Int,
```

Comment:
Instead of `hcIndex` could we just use `houseConsignment.sequenceNumber`? (Bearing in mind that sequence numbers start from 1 and not 0)